ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -402,6 +403,27 @@ def test_ipa_healthcheck_dna_plugin_returns_warning_pagure_issue_60(self):
         for check in data:
             assert check["result"] == "SUCCESS"
 
+    def test_ipa_healthcheck_log_rotate_file_exist_issue35(self):
+        """
+        This test checks if log rotation has been added
+        for ipa-healthcheck tool so that logs are rotated
+        in /var/log/ipa/healthcheck folder.
+        This is a testcase for below pagure issue
+        https://github.com/freeipa/freeipa-healthcheck/issues/35
+        """
+        msg = "error: /etc/logrotate.d/ipahealthcheck:"
+        tasks.uninstall_packages(self.master, HEALTHCHECK_PKG)
+        assert not self.master.transport.file_exists(
+            HEALTHCHECK_LOG_ROTATE_CONF
+        )
+        tasks.install_packages(self.master, HEALTHCHECK_PKG)
+        assert self.master.transport.file_exists(HEALTHCHECK_LOG_ROTATE_CONF)
+        cmd = self.master.run_command(
+            ["logrotate", "--debug", HEALTHCHECK_LOG_ROTATE_CONF]
+        )
+        assert cmd.returncode == 0
```

Comment:
This line is not needed as run_command is using raiseonerr=True by default and would raise an exception if the returncode is not 0.

Reply:
Replacing single quotes with double quotes was not done by pylint. Revered the change.
Also removed assert statement here.